_posts/2021-03-02-spring-2021-meeting-announced.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "PyHC Spring 2021 Meeting Dates Announced."
+author: jibarnum
+---
+
+PyHC serves as a community knowledge base for performing heliophysics research in Python, aiming to provide a variety of tutorials and resources, a list of useful packages, general discussion, and advice. Twice yearly, PyHC meets in person. These meetings are open to anyone, from individuals already utilizing Python heavily in developing open-source heliophysics packages to those who are just getting into using Python and open source development. 
+
+**PyHC's first meeting of 2021 will be held remotely on Zoom. The meeting will run Monday, May 10th, 2021 - Thursday, May 13th, 2021, from 9 AM - 11 AM MT each day.** We’ll hold tutorial, hackathon, and core project update sessions, as well as a couple other discussions that are relevant to the community. See [the full agenda](https://docs.google.com/spreadsheets/d/1zyakyCm6tnahqjHZHZchYxPrq8mf5FYIY_U3bvIpHNg/edit#gid=0) for dates/times of the various sessions. 
+
+The meeting's web page is coming soon, and the link will be posted here when it's live. **Registration, which is free, but required, can be found [here](https://docs.google.com/forms/d/1tk9uQTm9TzwNV8jle3QCg8IZkhZsQjLKMQDN-02a4IY/edit?usp=sharing).** The meeting's web page will also have a link to registration, as well as the Zoom telecon information. If you’d like to join our mailing list to learn more information about the meeting and other upcoming events, please see the [PyHC Contact page](http://heliopython.org/contact/) for instructions. We also have a [Riot chat group]( https://riot.im/app/#/room/#heliopython:openastronomy.org
+) where we discuss various PyHC issues/topics, as well as discussing other Python-related questions.
```

Comment:
The newline before this paren seems like a mistake, but GitHub's rich diff keeps the sentence on one line so it looks like it isn't consequential. I can't test this locally in my browser right now because the feature branch is in your personal GitHub instead of the heliophysicsPy one. I tried cloning your repo but I got a build error that I don't get with our repo and I didn't feel like spending any more time on it.

LGTM so long as that newline doesn't actually render on the site. 👍

Reply:
Ah, sorry about that - had worked off my cloned branch in the past and did that here out of habit. I'll change up how I do that in the future. :) 

Indeed, that newline shouldn't be there. I just tested it and it looks like it does *not* render on the site. That being the case, will go ahead and merge.